src/python_testing/TC_AVSM_2_16.py

```diff
@@ -178,6 +178,44 @@ async def test_TC_AVSM_2_16(self):
         aAudioRefCount = aAllocatedAudioStreams[0].referenceCount
 
         self.step(4)
+        # Ensure privacy modes are not enabled before issuing WebRTC
+        # ProvideOffer command
+        self.privacySupport = (aFeatureMap & cluster.Bitmaps.Feature.kPrivacy) > 0
+        if self.privacySupport:
+            result = await self.write_single_attribute(attr.SoftLivestreamPrivacyModeEnabled(False), endpoint_id=endpoint)
+            asserts.assert_equal(result, Status.Success, "Error when trying to write SoftLivestreamPrivacyModeEnabled")
+            logger.info(f"Tx'd : SoftLivestreamPrivacyModeEnabled{False}")
+
+            softLivestreamPrivMode = await self.read_single_attribute_check_success(
+                endpoint=endpoint, cluster=cluster, attribute=attr.SoftLivestreamPrivacyModeEnabled
+            )
+            asserts.assert_false(softLivestreamPrivMode, "SoftLivestreamPrivacyModeEnabled should be False")
+
+            result = await self.write_single_attribute(attr.SoftRecordingPrivacyModeEnabled(False), endpoint_id=endpoint)
+            asserts.assert_equal(result, Status.Success, "Error when trying to write SoftRecordingPrivacyModeEnabled")
+            logger.info(f"Tx'd : SoftRecordingPrivacyModeEnabled{False}")
+
+            softRecordingPrivMode = await self.read_single_attribute_check_success(
+                endpoint=endpoint, cluster=cluster, attribute=attr.SoftRecordingPrivacyModeEnabled
+            )
+            asserts.assert_false(softRecordingPrivMode, "SoftRecordingPrivacyModeEnabled should be False")
+
+        if await self.attribute_guard(endpoint=endpoint, attribute=attr.HardPrivacyModeOn):
+            # For CI: Use app pipe to simulate physical privacy switch being turned on
+            # For manual testing: User should physically turn on the privacy switch
+            if self.is_pics_sdk_ci_only:
+                self.write_to_app_pipe({"Name": "SetHardPrivacyModeOn", "Value": False})
```

Comment:
I've approved, but you also need to include the -app-pipe in the CI instruction block at the top